pacman/utilities/algorithm_utilities/element_allocator_algorithm.py

```diff
@@ -1,12 +1,9 @@
-from six import add_metaclass
 import math
 
 from pacman.model.resources import ElementFreeSpace
 from pacman.exceptions import PacmanElementAllocationException
-from spinn_utilities.abstract_base import AbstractBase
 
 
-@add_metaclass(AbstractBase)
```

Comment:
What was the reason for removing this? I wouldn't expect this class to be instantiated directly.

Reply:
but it is now. as i needed a key space allocation tracker. Not a malloc key allocator or the top level stuff.